### PR TITLE
Homepage updates

### PR DESCRIPTION
**Pull request by tw15egan**
Closes https://github.com/carbon-design-system/carbon-website/issues/724

- Adjusts margins on FeaturedTiles as well as ArticleTiles (Both have a margin on mobile now)
- New images on FeaturedTiles for mobile
- Changed some font styles

- Removed a `console.log` I found

### CODE REVIEW

**Comment by jeanservaas** [comment]
Looking so much better -- couple things:


- Can we change the margin bottom between "Start Dev'ing" and "Start Designing" to 16px instead of 24px?

image


- Can we add some percentage of right padding to these cards too (we can kill it at mobile)

image


- Awesome that you made this change to the Resources page too, I'm just wondering if we can fix the stacking arrangement there... i.e. under Featured, why are there two cards on top and three on the bottom... also, below in articles... they should also go to three columns like on the homepage... for some reason they are staying at two

image

- I would like to give you new images for some of these... the ones under Featured are really narrow, not the right aspect ratio etc.

- Please change the color of the author / date to Gray 70 only on the Resources page since it is on a white card not a dark one (565656)

image

**Comment by jeanservaas** [comment]
Right margins look great! and cards are stacking right in Resources now!

- Resource cards accidentally got a margin bottom too:

image

image

- Also, just to double check (are Author and date Gray 70? They look same color as the article title)

- Also 2, not sure why this is happening but these imgs all appear to have a light gray rule at the top of them in -- it's in Firefoxy and Chrome.

**Comment by tw15egan** [comment]
@jeanservaas updated 👍

File: src/components/ClickableTile/clickable-tile.scss [hunk 1]
```diff
@@ -42,59 +59,51 @@ a.tile--article.tile--dark:hover {
   border-bottom: none;
 }
 
+.tile__img span {
+  height: 100%;
+}
+
 .tile--article .tile__img img {
   width: 100%;
+  object-fit: cover;
```
Comment (alisonjoseph) [inline src/components/ClickableTile/clickable-tile.scss]: I'm not sure this works in IE11, so worth double checking how things look there

**Comment by jeanservaas** [comment]
@tw15egan 

I am almost ready to approve, just getting you new images for resource page and one other comment:

One last comment:
- I think we can hav right padding at Medium breakpoint and mobile too -- the only place where I would kill it it as Large (1056) breakpoint -- like you did. Think it can go a little less at mobile 25%.

image

**Comment by jeanservaas** [comment]
One last thing, Mike asked that we put this guy in instead of typing guy for the hackathon image on both homepage and resources

[Uploading homepage_03_revised.png.zip…]()

**Pull request merged**